nerfstudio/data/dataparsers/nerfstudio_dataparser.py

```diff
@@ -296,16 +292,23 @@ def _generate_dataparser_outputs(self, split="train"):
             applied_scale = float(meta["applied_scale"])
             scale_factor *= applied_scale
 
+        if "use_alpha_channel" in meta:
```

Comment:
I think this can be removed from the dataparser and the code would always load the alpha-channel if present (only when alpha_color is set it will blend before training - not sure if it will be necessary though).